### DISCUSSION TITLE
Feature request: Setting up the default reminder.

### DISCUSSION THREAD

**Discussion by lvpha** [bitfireAT/icsx5#48] [Ideas]
Hey everyone, 

Can you please add a feature to add or replace the valarm value in a calendar? The default reminder in google Cal is 3 mins and can be customized. But, unfortunately none of apps available provide a feature to customize the default reminder. 

Imagine a scenario when events are back to back, and the default reminder is 1h before the event. And you get confused by notifications. 

Thanks

**Comment by rfc2822** [comment]
Hi, this should be covered by #1.

**Reply by skjnldsv** [reply]
Hey Ricki!
It seems we're just missing the setting for all-day events :)

Workaround for now; I am able to enter a negative value like `-540` for the reminder to be at 9:00 on the morning
